python/Ganga/GPIDev/Lib/File/LocalFile.py

```diff
@@ -224,10 +223,26 @@ def remove(self):
 
         return
 
+    def copyTo(self, targetPath):
+        """
+        Copy a the file to the local storage using the get mechanism
+        Args:
+            targetPath (str): Target path where the file is to copied to
+        """
+        shutil.copy(path.join(self.localDir, self.namePattern), path.join(targetPath, self.namePattern))
```

Comment:
I think I disagree. The baseclass should call a generic method which will get the file if needed and will work with GoogleFile and LCGFile and etc. Only the files which know what to do otherwise can overload this.